pulp_container/app/modelresource.py

```diff
@@ -0,0 +1,99 @@
+from import_export import fields, widgets
+from pulpcore.plugin.importexport import QueryModelResource, BaseContentResource
+
+from pulp_container.app.models import Blob, Manifest, ManifestListManifest, Tag
+
+
+class BlobResource(BaseContentResource):
+    """
+    Resource for import/export of blob entities
+    """
+
+    def set_up_queryset(self):
+        """
+        :return: Blobs specific to a specified repo-version.
+        """
+        return Blob.objects.filter(pk__in=self.repo_version.content)
+
+    class Meta:
+        model = Blob
+        import_id_fields = model.natural_key_fields()
+
+
+class ManifestResource(BaseContentResource):
```

Comment:
Were not you planning to also add a separate ManifestListResource?

Reply:
I had it, but then with the extra fields on the join table, we need a separate resource for that one, and the need to serialize the ManifestLists separately vanished.

Reply:
nice, I have not realized that `` the need to serialize the ManifestLists separately vanished``